src/controllers/api.js

```diff
@@ -23,6 +23,7 @@ apiController.loadConfig = function (req, callback) {
 	config.browserTitle = validator.escape(String(meta.config.browserTitle || meta.config.title || 'NodeBB'));
 	config.titleLayout = (meta.config.titleLayout || '{pageTitle} | {browserTitle}').replace(/{/g, '&#123;').replace(/}/g, '&#125;');
 	config.showSiteTitle = meta.config.showSiteTitle === 1;
+	config.showFullnameAsDisplayName = meta.config.showFullnameAsDisplayName === 1;
```

Comment:
Unless I am mistaken, `=== 1` is discouraged here, add property to `defaults.json` instead and it should fall back gracefully in `meta.config`.

Reply:
It's added here to expose this value client side as boolean?

Reply:
Ah :+1: